Storybookで"../src/styles/global.scss"を解決することができない

# 概要
`ModuleNotFoundError: Module not found: Error: Can't resolve '../src/styles/globals.scss' in '/home/reohakuta/repos/shikosai32-front/.storybook'`
`main`ブランチの最新のコミットで`yarn run sb`を実行した場合でも同様のエラーが発生する。
# 全文
```shell
reohakuta in shikosai32-front on   feat/17-add_button
❯  yarn run sb
yarn run v1.22.17
$ start-storybook -p 6006
info @storybook/react v6.5.10
info
info => Loading presets
WARN Failed to load static files, no such directory: /home/reohakuta/repos/shikosai32-front/public
WARN Make sure this directory exists, or omit the -s (--static-dir) option.
info Addon-docs: using MDX1
info => Using PostCSS preset with postcss@8.4.16
info => Using default Webpack5 setup
<i> [webpack-dev-middleware] wait until bundle finished
10% building 0/1 entries 0/0 dependencies 0/0 modules
info => Using cached manager
21% building 3/15 entries 2617/2800 dependencies 352/557 modules
🌼 daisyUI components 2.24.0  https://github.com/saadeghi/daisyui
  ✔ Including:  base, components, themes[1], utilities

70% sealing plugins DocGenPluginDeprecationWarning: 'createIdentifier' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your tran
DeprecationWarning: '' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createLiteral' has been deprecated since v4.0.0. Use `factory.createStringLiteral`, `factory.createStringLiteralFromNode`, `factory.createNumericLiteral`, `factory.c
DeprecationWarning: 'createBinaryExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead
DeprecationWarning: 'createExpressionStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context inst
DeprecationWarning: 'createPropertyAssignment' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context inste
DeprecationWarning: 'createObjectLiteralExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context
DeprecationWarning: 'createFalse' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createNull' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createTrue' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createExpressionStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context inst
DeprecationWarning: 'createTypeOfExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead
DeprecationWarning: 'createElementAccessExpression' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context
DeprecationWarning: 'createIfStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createBlock' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createVariableDeclaration' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context inst
DeprecationWarning: 'createCatchClause' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
DeprecationWarning: 'createTryStatement' has been deprecated since v4.0.0. Use the appropriate method on 'ts.factory' or the 'factory' supplied by your transformation context instead.
99% done plugins webpack-hot-middlewarewebpack built preview aa93cc81feb1f1237318 in 6653ms
ModuleNotFoundError: Module not found: Error: Can't resolve '../src/styles/globals.scss' in '/home/reohakuta/repos/shikosai32-front/.storybook'
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/Compilation.js:2016:28
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/home/reohakuta/repos/shikosai32-front/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/home/reohakuta/repos/shikosai32-front/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /home/reohakuta/repos/shikosai32-front/node_modules/webpack/lib/NormalModuleFactory.js:975:5
resolve '../src/styles/globals.scss' in '/home/reohakuta/repos/shikosai32-front/.storybook'
  using description file: /home/reohakuta/repos/shikosai32-front/package.json (relative path: ./.storybook)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/reohakuta/repos/shikosai32-front/package.json (relative path: ./src/styles/globals.scss)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.js doesn't exist
      .jsx
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.jsx doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.ts doesn't exist
      .tsx
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.tsx doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.json doesn't exist
      .cjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss.cjs doesn't exist
      as directory
        /home/reohakuta/repos/shikosai32-front/src/styles/globals.scss doesn't exist

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

info => Loading presets
error Command failed with exit code 1.
```

--- a/.storybook/preview.js
+++ b/.storybook/preview.js
@@ -1,6 +1,6 @@
 import * as NextImage from 'next/image';
 import 'tailwindcss/tailwind.css';
-import '../src/styles/globals.scss';
+import '../src/styles/global.scss';
 
 const OriginalNextImage = NextImage.default;
 

